CHANGELOG.md

```diff
@@ -2,6 +2,17 @@
 
 All notable changes to this project will be documented in this file.
 
+## [[8.0.0]](https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/releases/tag/v8.0.0) 2023
+
+### New Features
+* Support Gramine framework for TEE tasks.
```

Comment:
Do we want to be more explicit ? This is just a modification in ITs to support new TEE tags.
There is no Gramine impact in blockchain adapter source code.

Reply:
Here it is https://github.com/iExecBlockchainComputing/iexec-blockchain-adapter-api/pull/72/commits/d467beeb0606c686ca0ba4e03e1a6d2460437f8c